pcsx2/gui/Panels/VideoPanel.cpp

```diff
@@ -43,7 +43,7 @@ Panels::FramelimiterPanel::FramelimiterPanel( wxWindow* parent )
 	m_text_BasePal		= CreateNumericalTextCtrl( this, 7 );
 
 	m_spin_NominalPct	->SetRange( 10,  1000 );
-	m_spin_SlomoPct		->SetRange(  1,  1000 );
+	m_spin_SlomoPct		->SetRange(  5,  1000 );
```

Comment:
Why was the range changed?

Reply:
I gave already an example but to reiterate, it doesn't allow lower than 5 as its input.
1.6 stable + 1.7 dev build before apply:
![image](https://user-images.githubusercontent.com/24227051/91488050-050ed380-e8af-11ea-9be8-8153d8de47c3.png)
After apply:
![image](https://user-images.githubusercontent.com/24227051/91488087-0dffa500-e8af-11ea-867e-212d8d58ea33.png)

Reply:
So if they changed the range when someone puts for example 1 it will automatically change to the preferred 5 already.
Or if they use the arrow keys it will only go to 5 as lowest number. Not sure if it's possible to force it do frame per frame instead of a slow motion (besides the point for this PR)